src/axom/inlet/Table.cpp

```diff
@@ -65,110 +65,88 @@ std::shared_ptr<Table> Table::addTable(const std::string& name,
   return currTable;
 }
 
```

Comment:
Side note: I'm starting to think that Axom should have a proper `Path` class that both `inlet` and `sidre` could use.
It can have utility functions like returning the parent, stem, etc...

Perhaps it can live in `core`?

Reply:
Why would it not live in sidre since its used for sidre paths?

Reply:
I suppose it could live in Sidre...
I was thinking of a path as just a delimited string -- without implying any sidre class structure.

Also, presumably paths in inlet can be different from those in sidre.

Reply:
That's true. core it is.